ambry-server/src/integration-test/java/com.github.ambry.server/MockCluster.java

```diff
@@ -25,6 +25,8 @@
 import com.github.ambry.notification.BlobReplicaSourceType;
```

Comment:
Changes in this class are mostly a re-org of existing code to avoid repeating code. The functions to track updates are new